.settings/com.google.gdt.eclipse.core.template.prefs

```diff
@@ -1,4 +0,0 @@
-eclipse.preferences.version=1
```

Comment:
Yeah due to some miss configurations of the project,I had to change these files too. That's why I changed these files.It has nothing to do with unused imports.